contracts/GovHub.template

```diff
@@ -300,7 +301,7 @@ contract GovHub is System, IParamSubscriber {
     } else if (block.number > proposal.endBlock + executingPeriod) {
       return ProposalState.Expired;
     } else {
-      return ProposalState.Succeeded;
+      return ProposalState.Succeeded; //@openissue
```

Comment:
I would recommend applying a timeLock here. most DAO implementations allow participants some time to react between a new proposal acceptance and actual execution. I no reason we will not do the same

Reply:
I strongly agree. The duration length is debatable, but imo proposals should not be enactable for 48 hours at minimum.

The only reason not to do so is if the governance members need to take quick action to stabilize/protect the system, but I can't think of any situations in which the governance members would have to act as a safety council to prevent massive loss of funds in our current design.